src/common/subgraphMapper.ts

```diff
@@ -190,50 +155,54 @@ export const mapOrdersArrayToInterface = (response: any): Order[] | undefined =>
 //////////////////////    POSITION    //////////////////////////
 ////////////////////////////////////////////////////////////////
 
-export const mapSinglePositionToInterface = (response: any): Position | undefined => {
+export const mapSinglePositionToInterface = (
+  response: any,
+  depositCollateral?: DepositCollateral[] | undefined,
+): Position | undefined => {
   if (response === null) return undefined;
   try {
     return {
       id: response.id,
       market: response.market ? mapSingleMarketToInterface(response.market) : undefined,
-      user: response.user ? mapSubgraphResponseToAccountInterface(response.user) : undefined,
+      user: response.user ? mapSubgraphResponseToWalletInterface(response.user) : undefined,
       isLong: response.isLong,
       positionCollateral: response.positionCollateral,
       positionSize: response.positionSize,
       avgPrice: response.avgPrice,
-      avgPriceDec: response.avgPriceDec,
-      lastCumulativeFee: response.lastCumulativeFee,
+      formattedAvgPrice: formatEther(response.avgPrice),
       status: response.status,
+      snxAccount: response.snxAccount.id,
       txHash: response.txHash,
       liquidationTxHash: response.liquidationTxHash,
       closingPrice: response.closingPrice,
+      formattedClosingPrice: formatEther(response.closingPrice),
```

Comment:
`formatEther` might dump if `undefined` is passed to it. Please test @Akhilleshgoswami @iamamitkumar0512

Reply:
sure

Reply:
done